the_iron_yelp/restaurants1.rb

```diff
@@ -0,0 +1,79 @@
+
+require 'json'
+class Restaurant
+  attr_reader :name, :address, :website, :hours, :price, :cuisines, :ratings
+
+  
+
+  def initialize(name, address, website, hours, price, cuisines, ratings)
+    @name = name
+    @address = address
+    @website = website
+    @hours = hours
+    @price = price
+    @cuisines = cuisines
+    @ratings = ratings
+  end
+  
+  def self.all 
+    restaurants_file = File.read "restaurants.json"
+    restaurants_array = JSON.parse restaurants_file 
+   
+    restaurants = restaurants_array.map do |restaurant|
+      
+       Restaurant.new(restaurant["name"], restaurant["address"],
+                  restaurant["website"], restaurant["hours"], 
+                  restaurant["price"], restaurant["cuisines"],
+                  restaurant["ratings"]) 
+   
+      end
+
+
+      return restaurants
+   end 
+  
+  def cheap?
+    @price.max <=10
+  end  
+
+  def self.cheap
+     self.all.select{|r| r.cheap?} 
+  end
+
+  def to_s
+    "#{@name}"
```

Comment:
that's redundant, since @name is a string, its interpolated string value will be the same string